src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1217,6 +1217,10 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
             throw new LogicException('Annotations cannot be enabled as the Doctrine Annotation library is not installed.');
         }
 
+        if (!class_exists('Doctrine\Common\Cache\ApcCache')) {
```

Comment:
I would check the existence of the interface rather than loading a particular implementation (which will not be the one we will use).